greenpeace/planet4-master-theme
# PLANET-7511 Migrate Articles block to Posts List block

Ref. https://jira.greenpeace.org/browse/PLANET-7511

### Summary

Articles block migration to post List block

---

### Testing
1. Run this PR in your local environment.
2. Add an Articles block to a post or page. You can also directly copy/paste block code snippet below in the code editor:
```

<div class="wp-block-planet4-blocks-articles"><div data-hydrate="planet4-blocks/articles" data-attributes="{&quot;article_heading&quot;:&quot;Related Articles&quot;,&quot;articles_description&quot;:&quot;&quot;,&quot;article_count&quot;:3,&quot;tags&quot;:[],&quot;posts&quot;:[],&quot;post_types&quot;:[],&quot;read_more_text&quot;:&quot;Load more&quot;,&quot;read_more_link&quot;:&quot;&quot;,&quot;button_link_new_tab&quot;:false,&quot;ignore_categories&quot;:false}"></div></div>



<div class="wp-block-planet4-blocks-articles"><div data-hydrate="planet4-blocks/articles" data-attributes="{&quot;article_heading&quot;:&quot;Articles block with Select tags&quot;,&quot;articles_description&quot;:&quot;Test description&lt;br&gt;Tags: Climate, Consumption, Oceans&quot;,&quot;article_count&quot;:4,&quot;tags&quot;:[6,8,26],&quot;posts&quot;:[],&quot;post_types&quot;:[],&quot;read_more_text&quot;:&quot;Load more&quot;,&quot;read_more_link&quot;:&quot;&quot;,&quot;button_link_new_tab&quot;:false,&quot;ignore_categories&quot;:false}"></div></div>



<div class="wp-block-planet4-blocks-articles"><div data-hydrate="planet4-blocks/articles" data-attributes="{&quot;article_heading&quot;:&quot;Articles block with Post types&quot;,&quot;articles_description&quot;:&quot;test description&lt;br&gt;post types: story, Press Release&quot;,&quot;article_count&quot;:2,&quot;tags&quot;:[],&quot;posts&quot;:[],&quot;post_types&quot;:[16,14],&quot;read_more_text&quot;:&quot;Load more&quot;,&quot;read_more_link&quot;:&quot;&quot;,&quot;button_link_new_tab&quot;:false,&quot;ignore_categories&quot;:false}"></div></div>



<div class="wp-block-planet4-blocks-articles"><div data-hydrate="planet4-blocks/articles" data-attributes="{&quot;article_heading&quot;:&quot;Articles block with Manual override posts &quot;,&quot;articles_description&quot;:&quot;Test description&lt;br&gt;Manual override Posts&quot;,&quot;article_count&quot;:5,&quot;tags&quot;:[],&quot;posts&quot;:[256,260,263,1107,1111,1113],&quot;post_types&quot;:[],&quot;read_more_text&quot;:&quot;Load more&quot;,&quot;read_more_link&quot;:&quot;&quot;,&quot;button_link_new_tab&quot;:false,&quot;ignore_categories&quot;:false}"></div></div>



<div class="wp-block-planet4-blocks-articles"><div data-hydrate="planet4-blocks/articles" data-attributes="{&quot;article_heading&quot;:&quot;&quot;,&quot;articles_description&quot;:&quot;empty title&quot;,&quot;article_count&quot;:3,&quot;tags&quot;:[],&quot;posts&quot;:[],&quot;post_types&quot;:[],&quot;read_more_text&quot;:&quot;Load more&quot;,&quot;read_more_link&quot;:&quot;&quot;,&quot;button_link_new_tab&quot;:false,&quot;ignore_categories&quot;:false}"></div></div>

```
3. Run the migrator command: `npx wp-env run cli wp p4-run-activator`
4. Check whether the migration worked as expected.
5. To reset the environment database back to the latest default content and repeat the tests, run: `npm run db:reset`.
